internal/nvmeof/controller/controllerserver.go

```diff
@@ -293,6 +294,78 @@ func (cs *Server) ControllerUnpublishVolume(
 	return &csi.ControllerUnpublishVolumeResponse{}, nil
 }
 
+// ControllerModifyVolume modifies the volume's QoS parameters.
+func (cs *Server) ControllerModifyVolume(
+	ctx context.Context,
+	req *csi.ControllerModifyVolumeRequest,
+) (*csi.ControllerModifyVolumeResponse, error) {
+	volumeID := req.GetVolumeId()
+	params := req.GetMutableParameters()
+
+	log.DebugLog(ctx, "ControllerModifyVolume: volumeID=%s, params=%+v", volumeID, params)
```

Comment:
This should  be logged in the Server.go please add this new RPC to https://github.com/ceph/ceph-csi/blob/fd202c7cf1a9619d528351769c108a41b17480f5/internal/csi-common/utils.go#L210